configure

```diff
@@ -67,6 +67,13 @@ save_configuration
 
 freshen_chef_repo
 
-cd chef_repo
+pushd chef_repo
 wd="$(pwd)"
-exec cinc-solo -c "${wd}/.chef/solo.rb" -E "$CHEF_ENVIRONMENT" -j "${wd}/${CHEF_SOLO_FILE}"
+cinc-solo -c "${wd}/.chef/solo.rb" -E "$CHEF_ENVIRONMENT" -j "${wd}/${CHEF_SOLO_FILE}"
+
+# Write date + commit SHA to a changelog
+echo "$( date "+%Y-%m-%d %H:%M:%S"):  $(git rev-parse --short HEAD)" >> "../CHEF_CHANGELOG.txt"; 
+popd 
+
+# Remove chef repository after configuration
+sudo rm -r chef_repo
```

Comment:
Git normally write-protects packfiles in the repository to prevent them from being interfered with unintentionally. As a side effect removing git repositories usually requires `-f`. Have you been able to avoid that here somehow?

Reply:
You can always work around things in linux with the `sudo` command :rofl: .